src/app/server/CommissioningWindowManager.cpp

```diff
@@ -390,6 +393,16 @@ CHIP_ERROR CommissioningWindowManager::OpenEnhancedCommissioningWindow(Seconds32
     return err;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+CHIP_ERROR CommissioningWindowManager::OpenJointCommissioningWindow(Seconds32 commissioningTimeout, uint16_t discriminator,
```

Comment:
I think we should add the corresponding unit tests for OpenJointCommissioningWindow inside src/controller/tests/TestCommissioningWindowOpener.cpp

Reply:
Added a unit test, TestCommissioningWindowOpener, for OpenJointCommissioningWindow in the latest commit.